src/appleseed-max-impl/appleseedinteractive/appleseedinteractive.h

```diff
@@ -95,6 +96,8 @@ class AppleseedInteractiveRender
     void AbortRender() override;
 
     void update_camera_object(INode* camera);
+    void assign_material(Mtl* mtl, INode* node);
```

Comment:
If `node` is the "receiver" of the material, I would suggest to put it in first position.